.github/workflows/build.yml

```diff
@@ -31,27 +31,30 @@ jobs:
           apt install -y libboost-all-dev thrift-compiler libthrift-dev
 
       - name: Initialize Project with West
-        working-directory: thrift-for-zephyr
+        working-directory: thrift-for-zephyr/..
         shell: bash
         run: |
           pip3 install -U west
-          west init -l .
+          west init zephyrproject
+          cd zephyrproject
+          cp ../thrift-for-zephyr/submanifests/99-thrift.yaml zephyr/submanifests/
           west update
-          pip3 install -r ../zephyr/scripts/requirements-base.txt
+          west zephyr-export
+          pip3 install -r zephyr/scripts/requirements.txt
 
       - name: Build and Run Testsuite
-        working-directory: thrift-for-zephyr
+        working-directory: thrift-for-zephyr/../zephyrproject
         shell: bash
         run: |
-          source zephyr-env.sh
```

Comment:
we should still source zephyr-env.sh, but let's make the CWD zephyr/ and refer to the thrift project via ../modules/lib/thrift. No need to set PATH either if using the upstream zephyr-env.sh